src/main/kotlin/tech/relaycorp/relaynet/keystores/CertificateStore.kt

```diff
@@ -8,43 +8,55 @@ import tech.relaycorp.relaynet.wrappers.x509.Certificate
 
 abstract class CertificateStore {
 
+    enum class Scope(val value: String) {
+        PDA("pda"),
+        CDA("cda")
+    }
+
     @Throws(KeyStoreBackendException::class)
-    suspend fun save(certificate: Certificate, chain: List<Certificate> = emptyList()) {
+    suspend fun save(
+        scope: Scope,
```

Comment:
In all these methods, I think the scope shouldn't be the first param because it's secondary. For example:

- `save(cert, scope, ...)`.
- `retrieveLatest(privateAddress, scope, ...)`
- And so on